docs/content/manual/manual.yml

```diff
@@ -705,17 +705,33 @@ sections:
       try to add a string to an object you'll get an error message and
       no result.
 
-      Please note that all numbers are converted to IEEE754 double precision
-      floating point representation. Arithmetic and logical operators are working
-      with these converted doubles. Results of all such operations are also limited
-      to the double precision.
-
-      The only exception to this behaviour of number is a snapshot of original number
-      literal. When a number which originally was provided as a literal is never
-      mutated until the end of the program then it is printed to the output in its
-      original literal form. This also includes cases when the original literal
-      would be truncated when converted to the IEEE754 double precision floating point
-      number.
+      Please note that jq uses IEEE754 double precision floating point
+      numbers for arithmetic computations, and that jq represents all
+      numbers internally as IEEE754 floats. The results of all
+      arithmetic operations are limited to double precision, but the
+      literal representation of integers and non-negative floats that
+      are presented to a jq program as literals without exponents ("e"
+      or "E") is retained and used for printing those numbers.
+
+      In this context, it should be noted that 'tonumber' retains a
+      snapshot of numeric strings that are exponent-free, and
+      `tostring` uses the snapshot representation, if available, as
+      illustrated by these examples:
+
+      "-1.000000000000000000000000000000000000000000000000000002" | tonumber
```

Comment:
Please add more 4 spaces indentation to make them look like code.

Reply:
Tx. Done.